tests/archive-test.sh

```diff
@@ -68,3 +68,11 @@ echo "ok cat-file"
 cd ${test_tmpdir}
 $OSTREE fsck
 echo "ok fsck"
+
+mkdir -p test-overlays
+date > test-overlays/overlaid-file
+$OSTREE commit ${COMMIT_ARGS} -b test-base --base test2 --owner-uid 42 --owner-gid 42 test-overlays/
+$OSTREE ls -R test-base > ls.txt
+cat ls.txt
+assert_streq "$(grep '42.*42' ls.txt | wc -l)" 2
```

Comment:
Could also check here that `ls.txt` has more than just 2 lines to make sure we did overlay on top of a base.